### PR TITLE
samples: multicore: hello_world: Align sample configuration

### PR DESCRIPTION
**Pull request by adamkondraciuk**
Align sample for testing purposes.

### CODE REVIEW

**Comment by coderbyheart** [comment]
Hell no!

**Comment by hubertmis** [comment]
Why are there two commits, each of them changing sdk-zephyr SHA to different revisions?

Do I understand correctly the overlays are a temporary solution, but later the power management nodes will be moved to soc's dts files?

File: samples/multicore/hello_world/power_management_app.overlay [hunk 1]
```diff
@@ -1,24 +1,25 @@
 /*
  * Copyright (c) 2024 Nordic Semiconductor ASA
  *
- * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ * SPDX-License-Identifier: Apache-2.0
```
Comment (zycz) [inline samples/multicore/hello_world/power_management_app.overlay:4]: Why d we change license here? Isn't it still part of sdk-nrf and we should use Nordic license anyway

File: samples/multicore/hello_world/power_management_app.overlay [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
-                         perm-read;
-                         perm-write;
-                         perm-execute;
-                         #address-cells = < 0x1 >;
-                         #size-cells = < 0x1 >;
-                         ranges = < 0x0 0x2fc00000 0x10000 >;
-                 };
-         };
- };
+		s2ram: s2ram {
```
Comment (zycz) [inline samples/multicore/hello_world/power_management_app.overlay:15]: The name might be a bit unfortunate. If someone doesn't know it means suspend to ram he might think it is some strange sram node. Maybe sus2ram to differentiate it from sram?

**Review by zycz — APPROVED** [review]

File: samples/multicore/hello_world/power_management_app.overlay (hunk 2 above)
Comment (hubertmis) [reply samples/multicore/hello_world/power_management_app.overlay:15]: S2RAM is a common abbreviation used all over computers (ok, mostly in Linux context :) ). I've never seen sus2ram.

**Review by gmarull — CHANGES_REQUESTED** [review]
power states should be defined at soc dts files, please, check upstream examples. This sample will also be removed sometime soon (moved upstream), its purpose is not to demo PM.

**Pull request closed without merging**